doc/specs/#658 - WinGet Download.md

```diff
@@ -0,0 +1,209 @@
+---
+author: Roy MacLachlan RDMaclachlan/roy.maclachlan@microsoft.com
+created on: 2023-02-09
+last updated: 2023-02-09
+issue id: 658
+---
+
+# Spec Title
+
+"For [#658](https://github.com/microsoft/winget-cli/issues/658)"
+
+## Abstract
+
+This spec describes the functionality and high-level implementation design for the downloading of applications using the Windows Package Manager.
+
+## Inspiration
+
+This is inspired by customer feedback, and a need for broader application deployments:
+* Customers want to share the installer with an offline device. 
+
+## Solution Design
+
+The Windows Package Manager will implement a new command `WinGet Download` that will offer customers with the ability to download a single package's installers. The new command will make use of the following optional parameter values:
+* `--id <Identifier>`
+* `--name <name>`
+* `--moniker <moniker>`
+* `--scope <scope>`
+* `--locale <locale>`
+* `--fileName <name>`
+* `-v, --version <version>`
+* `-s, --source <Source>`
+* `-a, --architecture <architecture>`
+* `-e, --exact`
+* `-o, --output <path>`
+* `-t, --type <installer type>`
+* `--ignore-security-hash`
+* `--accept-package-agreements`
```

Comment:
Do we need to accept package agreements just to download? I would have thought these were for installing.

Reply:
In enterprise scenarios, if an IT Admin downloads or installs an application they accept the EULA on-behalf of their customer. If an application has a EULA, they should accept this EULA when retrieving the applications. At time of Install the customer may not see it as the install could occur outside of the Windows Package Manager.